static/src/js/util/isPath.js

```diff
@@ -0,0 +1,15 @@
+import { castArray } from 'lodash'
+
+/**
+ * Returns true if the pathname matches one of the paths
+ * @param {string} pathname - The pathname to check.
+ * @param {string|array} paths - The path(s) to check against.
+ * @return {boolean}
+ */
+export const isPath = (pathname, paths) => castArray(paths).some((path = '') => {
+  const pathnameWithoutTrailingSlash = pathname.replace(/\/+$/, '')
+  const pathWithoutTrailingSlash = path.replace(/\/+$/, '')
+  return pathnameWithoutTrailingSlash === pathWithoutTrailingSlash
+})
+
+export default isPath
```

Comment:
Exporting as default but when its used you're using the named export (which I *think* is preferred)

Reply:
I think _preffered_ is really just a matter of opinion. We have an eslint rule thats enforcing this at the moment. We may just want to nix that and remove this from all utils.